.github/workflows/ecr_gc_docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Garbage Collect ECR Images
+
+on:
+  schedule:
+    # Runs every 30 minutes
+    - cron: "*/30 * * * *"
+  # Have the ability to trigger this job manually using the API as well
+  workflow_dispatch:
```

Comment:
is it green when you manually triggering it?

Reply:
Can't actually manually trigger it until we merge first unfortunately.

Would be a good use for a canary repository though

Reply:
I think we can add on pull_request temporally to test the script, and then before the merge remove the on pull_request trigger